deploy/canary/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM golang:latest
 MAINTAINER vmarmol@google.com
 
-RUN apt-get install -y git dmsetup
+RUN apt-cache update && apt-get clean && apt-get install -y git dmsetup
```

Comment:
you probably wanted the `apt-get clean` after the `apt-get install` to remove the .debs downloaded.

Reply:
ahh, ok